runelite-client/src/main/java/net/runelite/client/plugins/puzzlesolver/PuzzleSolverConfig.java

```diff
@@ -55,10 +59,70 @@ default boolean displayRemainingMoves()
 	@ConfigItem(
 		keyName = "drawDots",
 		name = "Draw dots instead of arrows",
-		description = "Draw dots increasing in size instead of arrows for the solution."
+		description = "Draw dots increasing in size instead of arrows for the solution.",
+		position = 2
 	)
 	default boolean drawDots()
 	{
 		return false;
```

Comment:
As discussed in the Discord at the least dots should probably be the default for sake of UX. Might be enough to change to true here. I don't know how defaults work in the config system - if this will change to true on next plugin start for existing users who have not manually flipped it. Or if it stays false for them. That's fine imo either way. If someone really liked arrows they can switch it off